anchore_engine/services/catalog/__init__.py

```diff
@@ -768,6 +768,14 @@ def handle_repo_watcher(*args, **kwargs):
                                 )
                                 raise
 
+                            if not catalog_impl.is_image_valid_size(new_image_info):
+                                localconfig = (
+                                    anchore_engine.configuration.localconfig.get_config()
+                                )
+                                raise Exception(
+                                    f"Image size of {new_image_info['compressed_size']} exceeds configured maximum of {localconfig.get('max_compressed_image_size')}"
```

Comment:
minor: We're using several different string interpolation methods across the code base, but I haven't seen many F strings...might want to use format or % for consistency?

Reply:
switched to concatenation to match the rest of the file